components/tcf/ui/src/FirstLayer/index.scss

```diff
@@ -60,29 +43,10 @@ $base-class-modal: '#sui-TcfFirstLayerModal';
   &-buttons {
     display: flex;
     justify-content: center;
-    margin-top: $m-xl;
-
-    #{$base-class}:not(#{$base-class}--isMobile) .sui-MoleculeNotification & {
-      @include media-breakpoint-up(s) {
-        flex-direction: column-reverse;
-        min-width: $maw-tcf-first-layer-buttons;
-        border-top: 0;
-        justify-content: flex-end;
```

Comment:
before the modal change, the buttons in desktop device were shown aligned to the right of, the fix should include that

Reply:
Solved